gdnative-core/src/init/macros.rs

```diff
@@ -88,6 +88,7 @@ macro_rules! godot_gdnative_init {
                 return;
             }
 
+            $crate::private::init_main_thread_id();
```

Comment:
The way we declare entry points has been changed in #1026. The function bodies can now be found in `gdnative-core/src/init/private.rs`. Please see if you can rebase your work on the latest `master`.

Reply:
fixed.